idd/Energy+.idd.in

```diff
@@ -40719,6 +40725,9 @@ ZoneHVAC:AirDistributionUnit,
        \key AirTerminal:DualDuct:ConstantVolume
        \key AirTerminal:DualDuct:VAV
        \key AirTerminal:SingleDuct:ConstantVolume:Reheat
+       \key AirTerminal:SingleDuct:ConstantVolume:FourPipeBeam
+       \key AirTerminal:SingleDuct:InletSideMixer
+       \key AirTerminal:SingleDuct:SupplySideMixer
```

Comment:
And while we're at it here, we should add a new \reference AirTerminalUnitNamess to all of these and use \object-List AirTerminalUnitNames in the next field here.